runtime-management/runtime-management-impl/src/main/java/io/cloudslang/runtime/impl/python/external/ResourceScriptResolver.java

```diff
@@ -40,12 +45,26 @@
         }
     }
 
+    private static String loadScriptFromResourceAsString(String resourceName) {
+        try (InputStream stream = ResourceScriptResolver.class.getClassLoader().getResourceAsStream(resourceName)) {
```

Comment:
Not an urgent issue, but some classloaders might return `null`. See here: https://docs.oracle.com/javase/6/docs/api/java/lang/Class.html#getClassLoader()

Maybe we should null check `ResourceScriptResolver.class.getClassLoader()` first

Reply:
Good catch!